tests/unit_tests/query_handler_runner/test_mock_query_handler_runner.py

```diff
@@ -40,6 +41,12 @@ def top_level_query_handler_context(mocked_temporary_bucketfs_location,
     return top_level_query_handler_context
 
 
+def normalize(queries: Union[str, List[str]]):
```

Comment:
Did you add this?

Reply:
Ok, for this specific case probably not harmful, but general it could be harmful. On the other side, I think we should also test the formatting of the queries. However, we might want to use multi line strings in the expected queries with cleandoc surrounded.

Reply:
Cleandoc will remove the trailing whitespace after `SELECT`.

Reply:
I replaced trailing spaces with `\x20`.